utils/images.ahk

```diff
@@ -27,3 +27,21 @@ GetCurrentMapPage() {
         }
     }
 }
+
+SearchUpgrade1() {
+	if (SearchImage("buttons\upgrades\1R", "*TransBlack", 250, 475, 1494, 499) or SearchImage("buttons\upgrades\1L", "*TransBlack", 250, 475, 1494, 499)) {
```

Comment:
There's no reason for the *TransBlack option here as the images don't have any black that should be made transparent.

Reply:
Replaced these with "".